UniversalAlgebra/GaloisConnections/Basic.lagda

```diff
@@ -23,14 +23,17 @@ In other terms, F is a left adjoint of G and G is a right adjoint of F.
 
 module GaloisConnections.Basic where
 
-open import Agda.Primitive          using    ( _⊔_ ;  Level )
+open import Agda.Primitive          using    ( _⊔_ ;  Level ; lsuc)
                                     renaming ( Set to Type )
 open import Relation.Binary.Bundles using    ( Poset )
 open import Relation.Binary.Core    using    ( REL ; Rel ; _⇒_ )
 open import Relation.Unary          using    ( _⊆_ ;  _∈_ ; Pred   )
 
-module _ {α α₁ α₂ : Level}(A : Poset α α₁ α₂)
-         {β β₁ β₂ : Level}(B : Poset β β₁ β₂)
+
+
+module _ {ℓ : Level}
```

Comment:
Why the collapse of all the levels?

Reply:
I was having trouble getting it to work in full generality.  I just wanted to get it to type-check and commit quickly because Galois connections is not a priority. I just did it as a fun exercise that I thought would be very quick and easy.

Now that it's working we might try to generalize to arbitrary levels.